File: Factory/Java/Rectangle.java
public class Rectangle implements Shape {
    @Override
    public void draw()
    {
        for(int i = 0; i < 3; i++)
        {
            for(int j = 0; j < 8; j++)
            {
                System.out.print("* ");
            }
            System.out.println("");
        }
    }
    
}

File: Factory/Python/Rectangle.py
from Shape import shape

class rectangle(shape):

    """ Esta clase hereda de la super clase Shape """

    def __init__(self, numLados):
        print("Rectangulo")
        self.numLados = numLados

    def draw(self):
        for i in range(0, self.numLados):
            for j in range(0, self.numLados*2):
                print("* ", end="")
            print(" ")
